lib/ontologies_linked_data/models/users/user.rb

```diff
@@ -18,6 +18,9 @@ class User < LinkedData::Models::Base
       attribute :role, enforce: [:role, :list], :default => lambda {|x| [LinkedData::Models::Users::Role.default]}
       attribute :firstName
       attribute :lastName
+      attribute :github_id, enforce: [:unique]
+      attribute :orcid_id, enforce: [:unique]
+      attribute :is_subscribe, enforce: [:existence], :default => lambda {|x| false}
```

Comment:
We don't need to enforce its existence: the default value is enough.
And add it in a separate PR